pages/glossary/foreground-colors-of-text.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Foreground Colors Of Text Nodes
+key: foreground-colors-of-text
+---
+
+All the colors of the pixels of a [visible](#visible) [text node](https://dom.spec.whatwg.org/#text) that change when the CSS `color` property is changed. This includes anti-aliased pixels.
```

Comment:
The colors of all the pixels of a [visible](#visible) [text node](https://dom.spec.whatwg.org/#text) that change color when the CSS `color` property is changed. This includes anti-aliased pixels.

Plus, I'm not sure if visible is needed for the definition.

Reply:
It is, white text on white background is not visible, but changes when the color property is changed.